doc/daal4py/contents.rst

```diff
@@ -21,7 +19,7 @@ Contents
 ########
 
 .. include:: note.rst
-   
+
```

Comment:
What linter did you run on the files? (Not that its a big deal, just curious to see if we are missing something)

Reply:
A text editor setting that removes trailing spaces in files upon saving.